slurm/install/install.py

```diff
@@ -429,7 +393,7 @@ def main() -> None:
     munge_key(settings)
 
     # runs either rhel.sh or ubuntu.sh to install the packages
-    run_installer(os.path.abspath(f"{args.platform}.sh"), args.mode)
+    run_installer(settings, os.path.abspath(f"{args.platform}.sh"), args.mode)
```

Comment:
This is problematic as script is expecting a relative path to slurm-pkgs. although in cluster-init we are cd ing to the directory, subprocess is running it from absolute path.

Reply:
Good point. For the actual scripts they are running, I added a cd $(realpath $(dirname $0)) at the root so that it is irrelevant from where it is invoked for now.